tools/tasks/compile.js

```diff
@@ -5,8 +5,8 @@ import settings from './settings';
 export default function compile() {
   const deferred = new Deferred();
 
-  const platform = process.platform === 'darwin'
-    ? '-f netcoreapp1.1'
+  const platform = process.platform === ('darwin' || 'win32')
+    ? '-f netcoreapp2.0'
     : '';
```

Comment:
The purpose of these lines of code was to only build/test .NET Core when on macOS.  This switches it to build only .NET Core on both macOS and Windows, which isn't the wanted behavior.  When on Windows it should build both .NET Core and the full-framework .NET version (as it was previously).